backend/app/utils/hls.py

```
@@ -0,0 +1,73 @@
+                f"-hls_segment_filename {output_dir}/{resolution}_%03d.ts {output_m3u8}"
+                )
+            else :
+                command = (
+                f"ffmpeg -hwaccel auto -i {input_file} -vf scale={settings['scale']} -c:a aac -ar 48000 "
+                f"-b:a {settings['audio_bitrate']} -c:v h264 -profile:v main -crf 20 "
+                f"-sc_threshold 0 -g 48 -keyint_min 48 -hls_time 10 -hls_playlist_type vod "
+                f"-b:v {settings['video_bitrate']} -maxrate {settings['video_bitrate']} -bufsize 2000k "
+                f"-hls_segment_filename {output_dir}/{resolution}_%03d.ts {output_m3u8}"
+                )
+            print(f"Running command: {command}")  # Debugging
+            process = subprocess.run(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
```

Ruff (0.15.15)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)